examples/opensora_pku/opensora/models/diffusion/latte/modeling_latte.py

```diff
@@ -1427,24 +1427,27 @@ def construct(
             # assume that mask is expressed as:
             #   (1 = keep,      0 = discard)
             # convert mask into a bias that can be added to attention scores:
-            #       (keep = +0,     discard = -ms.numpy.inf)
+            #       (keep = +0,     discard = -10000.0)
             attention_mask = attention_mask.unsqueeze(1)
-            attention_mask = ops.zeros(attention_mask.shape).masked_fill(~attention_mask, -ms.numpy.inf)
+            # do not fill in -10000.0 until entering MHA
+            # attention_mask = ops.zeros(attention_mask.shape).masked_fill(~attention_mask, -10000.0)
```

Comment:
Because torch code masked_fill with -10000.0 in this part: [See torch implementation](https://github.com/PKU-YuanGroup/Open-Sora-Plan/blob/main/opensora/models/diffusion/latte/modeling_latte.py#L343-L368).

Here I want to use this comment to explain the code difference between ms and torch. Torch will mask_fill before entering MHA, while MS will mask_fill until entering MHA. It's OK to just delete this comment.

The mask flip and broadcast is done inside MHA.

Reply:
FYI, those misleading comments were deleted.